src/battle.ts

```diff
@@ -626,6 +627,9 @@ class Side {
 	setAvatar(avatar: string) {
 		this.avatar = avatar;
 	}
+	setRating(rating: string) {
```

Comment:
I wouldn't make a setter for this. `setAvatar` exists as an external API, but unless someone needs to set `rating` externally, you can just set `rating` directly; it's defined in the same file, after all.